lib/ansible/modules/iptables.py

```diff
@@ -7,22 +7,21 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 DOCUMENTATION = r'''
 ---
 module: iptables
 short_description: Modify iptables rules
-version_added: "2.0"
+version_added: "3.0"
```

Comment:
this is version of ansible core in which the ENTIRE module was added, do not change